### PR TITLE
add xsky block csi charts to rancher

### PR DESCRIPTION
**Pull request by zhucan**
Signed-off-by: zhuc <zhucan@xsky.com>

### CODE REVIEW

File: charts/xsky/v2.0.1/questions.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+           - IfNotPresent
+           - Always
+           - Never
+       - variable: operator.image.repository
+         default: "xskydriver/csi-iscsi"
+         description: "XSKY operator image name"
+         type: string
+         label: XSKY Operator Image Name
+       - variable: operator.image.tag
+         default: "2.1.0"
```
Comment (Jason-ZW) [inline charts/xsky/v2.0.1/questions.yaml:35]: Is that version correct?  Maybe `2.0.1` not `2.1.0`?

**Review by Jason-ZW — CHANGES_REQUESTED** [review]
It's best to include content in `README.md` and it doesn't matter if you don't accept it.
Others LGTM.

File: charts/xsky/v2.1.0/templates/nodeplugin-daemonset.yaml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+          volumeMounts:
+            - name: plugin-dir
+              mountPath: {{ .Values.pluginsDir }}
+            - name: registration-dir
+              mountPath: /var/lib/kubelet/plugins_registry
+              mountPropagation: "Bidirectional"
+            - name: pods-mount-dir
+              mountPath: /var/lib/kubelet/pods
+              mountPropagation: "Bidirectional"
+		    - name: socket-dir
```
Comment (Jason-ZW) [inline charts/xsky/v2.1.0/templates/nodeplugin-daemonset.yaml]: Could you please modify the indent?

Comment (Jason-ZW) [inline charts/xsky/v2.1.0/templates/nodeplugin-daemonset.yaml]: Could you please change the indent?

Comment (zhucan) [reply charts/xsky/v2.1.0/templates/nodeplugin-daemonset.yaml]: I will do it out of day

File: charts/xsky/v2.0.1/questions.yaml (hunk 1 above)
Comment (zhucan) [reply charts/xsky/v2.0.1/questions.yaml:35]: README.md has been added

**Review by Jason-ZW — APPROVED** [review]

**Pull request merged**